tensorboard/plugins/profile/profile_plugin.py

```diff
@@ -142,65 +144,77 @@ def index_impl(self):
     for a specific tool "x" will have a suffix name TOOLS["x"].
     Example:
       log/
-        run1/
+        model1/
```

Comment:
Thanks @nfelt  The code actually searches for any subdirectories under root that called profile.
A run subdirectory was always created under the plugins/profile/ directory, for each profile runs. Note here the run means profile runs rather than model runs: when 1 model is running the user can capture a TPU profile for multiple times. Therefore, it makes sense to have the run subdirectories under plugins/profile. We expect the users to have root/model#/plugins/profile/run#, where root/model# is the model checkpoints. I think it is a expected user behavior when running cloud TPU models in GCE, and that's what #1290 is asking for.

Reply:
For the record - I misread the code here due to confusion about the terminology of the run subdirectories, this is fine and we're on the same page.